src/test/python/agents/script/sample_script.cfg

```diff
@@ -0,0 +1,16 @@
+[Locations]
+script_file = ./sample_script.py
+
+# Name of the bot in-game
+name = Sample Script
+
+[Details]
+# These values are optional but useful metadata for helper programs
+# Name of the bot's creator/developer
+developer = tarehart
+
+# Link to github repository
+github = https://github.com/tarehart/RLBotTwitchBroker
+
+# Programming language
+language = python
```

Comment:
Do these support parameters similar to bots?

Reply:
Nope. With bots, I believe the framework reads the config and feeds it to the bot in some callback / lifecycle type function. Scripts get no callbacks, they're on their own. There's nothing stopping them from reading their own config file, though.

I'm inclined to leave it the way it is, I like having loose coupling for these scripts.

Reply:
Cool